assets/themes/default/_forms.scss

```diff
@@ -144,7 +144,7 @@ form.new-post {
 
   & > * {
     flex-grow: 1;
-    max-width: 40%;
+    max-width: 60%;
```

Comment:
This bit here seems to make the _article meta_ less squashed than before on small screens, I think?

Before:
<img width="438" alt="Screenshot 2019-09-01 at 19 23 09" src="https://user-images.githubusercontent.com/45913977/64079905-1e0d9480-ccee-11e9-9433-7ea082f6f757.png">

After #662:

<img width="440" alt="Screenshot 2019-09-01 at 19 23 44" src="https://user-images.githubusercontent.com/45913977/64079909-2f56a100-ccee-11e9-8231-0e97fd267362.png">
 
May be worth replicating in #663 if you want...

Reply:
I'm not sure these lines apply on *article meta* section, as it doesn't have the `split` class too (it is used only for the license/tags part, not the rest). But I have something similar to that in my PR. Thanks anyway!